src/entities/Proposal/utils.ts

```diff
@@ -215,3 +216,7 @@ export function isProposalInCliffPeriod(grant: GrantAttributes) {
   const now = Time.utc()
   return !isOneTimePayment && Time.unix(grant.enacted_at).add(CLIFF_PERIOD_IN_DAYS, 'day').isAfter(now)
 }
+
+export function getProposalCategory(proposalType: ProposalType, proposalConfiguration: any): string | null {
+  return proposalType === ProposalType.Grant ? proposalConfiguration.category : null
```

Comment:
As this is used directly in the model. Can we validate the category we are returning is a valid category?